### PR TITLE
Basic 10.1 updates for Ele shaman

### PR DESCRIPTION
**Pull request by periodic**
### Description

I created a much larger PR with new functionality, but in the interest of small PRs I extracted the miminum that I think we can get away with to get the the Elemental Shaman analyzer to be modernized for 10.1. 

This includes only:

- Basic contributor updates.
- New spells in the tracking list.
- Fixes for tracking of Stormkeeper and Surge of Power buffs.

This is the first step in a larger overhaul that I am planning with @awildfivreld

Issue: https://github.com/WoWAnalyzer/WoWAnalyzer/issues/6096

### Testing

Basically you can compare this report the the old one.  Here are some examples of the differences:

#### Surge of Power now tracks affected casts accurately

It had no header and didn't catch all buffs.

Before:
image

After:
image

#### Stormkeeper catches all casts now

Before: 
image

After:
image

#### More cooldowns added to the ability list

Added:

- Earth Elemental
- Ancestral Guidance
- Spiritwalker's Grace
- Nature's Swiftness
- Ghost Wolf
- Gust of Wind

Updated:

- Corrected cooldown for talent choices on Astral Shift

image

### CODE REVIEW

File: src/analysis/retail/shaman/elemental/CONFIG.tsx [hunk 1]
```diff
@@ -51,7 +52,7 @@ export default {
   ),
   // A recent example report to see interesting parts of the spec. Will be shown on the homepage.
   exampleReport:
-    '/report/YgrNJWQpCwh4XBfj/5-Normal+The+Primal+Council+-+Kill+(4:15)/Blitzlimo/standard',
+    'http://localhost:3000/report/L1R2zVKY8ZDxCTNb/15-Mythic+Scalecommander+Sarkareth+-+Kill+(7:21)/Nerdshockz/standard',
```
Comment (emallson) [inline src/analysis/retail/shaman/elemental/CONFIG.tsx]:
```suggestion
    '/report/L1R2zVKY8ZDxCTNb/15-Mythic+Scalecommander+Sarkareth+-+Kill+(7:21)/Nerdshockz/standard',
```

omit the domain for this. viewers won't have `localhost:3000` up :)

**Review by emallson — CHANGES_REQUESTED** [review]
one small change to fix the example report link, otherwise this looks good 👍

File: src/analysis/retail/shaman/elemental/CONFIG.tsx (hunk 1 above)
Comment (periodic) [reply src/analysis/retail/shaman/elemental/CONFIG.tsx]: LOL, how did I miss that. :D

**Comment by periodic** [comment]
Fixed that part.  Also, I realized I forgot to stage a small change that actually fixed the display of Surge of Power's statistic.

**Review by emallson — APPROVED** [review]

**Pull request merged**